src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -298,17 +298,42 @@ public void Initialize()
             }
         }
 
-        public async Task PutEditVersioningCommandAsync(JsonOperationContext context, string databaseName, BlittableJsonReaderObject val)
+        public async Task<long> ModifyDatabaseExpirationBundle(TransactionOperationContext context, string name, BlittableJsonReaderObject configurationJson)
```

Comment:
Guys i wrote my code and then merged the changes made to commands. those hindsight comments are really annoying.